client/src/docs-ui/landing-hero-cta/__snapshots__/landing-hero-cta.spec.ts.snap

```diff
@@ -14,7 +14,7 @@ exports[`docs-landing-hero-cta Render logic should render 1`] = `
     <docs-internal-link class="scale-up-on-hover" href="/start/q/integration/vue">
       <img alt="Vue Icon" class="css-16ygk4v" src="/assets/integrations/vue.svg">
     </docs-internal-link>
-    <docs-internal-link class="scale-up-on-hover" href="/start/">
+    <docs-internal-link class="scale-up-on-hover" href="/start">
```

Comment:
Does this break anything?

Reply:
No -- in fact it fixes the "no such page `/start/`" error that comes up in the console when visiting the home page before selecting a framework.